dev/com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1.envVars/bnd.bnd

```diff
@@ -0,0 +1,73 @@
+#*******************************************************************************
+# Copyright (c) 2018,2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src    
+
+fat.project: true
+
+-dependson: \
+    build.changeDetector,\
+    com.ibm.ws.security.fat.common.jwt,\
+    com.ibm.ws.security.fat.common.mp.jwt,\
+    com.ibm.ws.security.mp.jwt.1.1_fat.commonTest
+    
+# Dependencies may be local bundles (e.g. com.ibm.websphere.javaee.servlet.3.1)
+#      or binaries from Artifactory (e.g. commons-logging:commons-logging)
+# For all project names that match the pattern "*_fat*", dependencies for junit,
+# fattest.simplicity, and componenttest-1.0 will be automatically added to the buildpath	
+-buildpath: \
+	com.ibm.websphere.javaee.annotation.1.2;version=latest,\
+	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
+	com.ibm.websphere.javaee.cdi.2.0;version=latest,\
```

Comment:
Should only need one of these.

A separate exercise we should go through (at some point) is cleaning up these bnd files for FAT projects because I imagine there are tons of superfluous packages included in these files at this point.